README.md

```diff
@@ -1,5 +1,13 @@
 # FS19_modROS
 
+## Landfill compactor mod
+
+The mod for the landfill compactor is available here https://mods2019portal.com/farming-simulator-2019-mods/forklifts-excavators/cat-836k-landfill-eiffage-v1.5.html.
```

Comment:
Hey Tom, 
(1) Can you elaborate more on how to download and install the compactor mod. There is a lot of Spam Download buttons on this page. Is it the External Download button?  A few screenshots would be helpful.
(2) Also can you add a description and/or screenshots of how to sell other vehicles

Reply:
Thanks